tinygrad/runtime/ops_gpu.py

```diff
@@ -49,6 +49,9 @@ def _copyout(self, x:np.ndarray):
     buf = cl.Buffer(CL.cl_ctxs[self._buf.device], cl.mem_flags.WRITE_ONLY | cl.mem_flags.USE_HOST_PTR, 0, hostbuf=x.data)
     mapped, event = cl.enqueue_map_buffer(CL.cl_queue[self._buf.device], buf, cl.map_flags.WRITE, 0, self.size, dtype=self.dtype.np, is_blocking=False)
     with mapped.base: cl.enqueue_copy(CL.cl_queue[self._buf.device], mapped, self._buf, is_blocking=True, wait_for=[event])
+  def _transfer(self, x):
+    if "gfx" in CL.cl_ctxs[x._buf.device].devices[0].name: # TODO: only on amd
```

Comment:
Err, this will just silently do nothing on non AMD?

Reply:
yea, currently it kinda does, thats why i gated it in lazy.py behind an envvar, im not sure what the best way to handle this is